src/Client/QueryFuzzer.cpp

```diff
@@ -905,11 +905,38 @@ void QueryFuzzer::fuzz(ASTPtr & ast)
                 select->where()->children.clear();
                 select->setExpression(ASTSelectQuery::Expression::WHERE, {});
             }
+            else if (!select->prewhere().get())
+            {
+                if (fuzz_rand() % 50 == 0)
+                {
+                    select->setExpression(ASTSelectQuery::Expression::PREWHERE, select->where()->clone());
```

Comment:
You can add other way also. If you have prewhere, move it to where. Otherwise where will be rare